bundles/org.openhab.binding.radiothermostat/src/main/java/org/openhab/binding/radiothermostat/internal/handler/RadioThermostatHandler.java

```diff
@@ -384,6 +387,13 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                         connector.sendCommand("rem_mode", "0", REMOTE_TEMP_RESOURCE);
                     }
                     break;
+                case MESSAGE:
+                    if (!BLANK.equals(cmdStr)) {
```

Comment:
Perhaps `UnDefType` could also be supported, so that `UnDefType.UNDEF` and`UnDefType.NULL`  would also clear the display?

Reply:
I tried that but UnDefType is a State and can't be sent via a Command. Sending null or empty string works (both present as empty string when doing Command.toString()). I updated the documentation to note sending either null or empty will clear the message.

Reply:
You are right, sorry.